packages/node/src/lib/changes/changes.ts

```diff
@@ -0,0 +1,43 @@
+import { AxiosInstance } from 'axios';
```

Comment:
Replace with `import axios from 'axios',`. It won't be used but it's necessay for types coming with WithHttp